src/main/asciidoc/_chapters/configuration.adoc

```diff
@@ -291,7 +291,7 @@ link:https://hadoop.apache.org/cve_list.html[CVEs] so we drop the support in new
 
 [cols="1,4*^.^", options="header"]
 |===
-| | HBase-1.4.x | HBase-1.6.x | HBase-1.7.x | HBase-2.3.x
+| | HBase-1.4.x | HBase-1.6.x | HBase-1.7.x | HBase-2.3.x and HBase-2.4.x
```

Comment:
I think we'd better have a separated column for 2.4.x? IIRC 2.3.x will be EOL soon after the final release of 2.3.7. Correct me if I'm wrong.

Reply:
Sure, we can do that